zebra-network/src/meta_addr.rs

```diff
@@ -247,6 +247,14 @@ pub enum MetaAddrChange {
         addr: SocketAddr,
         services: Option<PeerServices>,
     },
+
+    NewInitial {
```

Comment:
Nit: It might make it easier in the future to change code related to this if the `New*` variants are grouped together. I think you could move the `NewInitial` to the start of the `enum` so that it's the first variant. The same can also be done on the `match` statements in each method, and also for the `new_initial` constructor.